workflows/genome_annotation/annotation_helixer/Galaxy-Workflow-annotation_helixer-tests.yml

```diff
@@ -0,0 +1,73 @@
+- doc: Test outline for Helixer Workflow
+  job:
+    Input:
```

Comment:
The input and output labels are shown to the user in the workflow run form and in the workflow invocation and report. Could you rename these to be human friendly, that is replace the `_` with a normal whitespace character ` ` 
 and spell out genome instead of geno and peptides instead of pep ? If you do this it would need to be done in the test file and the workflow file.